### ISSUE TITLE
Missing commands, e.g., *-AtwsTicket

### ISSUE THREAD

**Issue by MitchellVanHorn** [officecenter/Autotask#24]
I've noticed commands that used to exist, e.g., Get-/Set-/New-AtwsTicket are no longer present in the 1.6.2.4 module. Hopefully this is caused by a fixable error in the module. Just in case, here is a full list of commands that existed in my cached 1.6.2.2, but do not exist for my new installations of 1.6.2.4

```
Function        Get-AtwsAccount                                    1.6.2.2    Autotask
Function        Get-AtwsAccountAlert                               1.6.2.2    Autotask
Function        Get-AtwsAccountNote                                1.6.2.2    Autotask
Function        Get-AtwsAccountToDo                                1.6.2.2    Autotask
Function        Get-AtwsActionType                                 1.6.2.2    Autotask
Function        Get-AtwsAdditionalInvoiceFieldValue                1.6.2.2    Autotask
Function        Get-AtwsAllocationCode                             1.6.2.2    Autotask
Function        Get-AtwsAttachmentInfo                             1.6.2.2    Autotask
Function        Get-AtwsBillingItem                                1.6.2.2    Autotask
Function        Get-AtwsBusinessLocation                           1.6.2.2    Autotask
Function        Get-AtwsBusinessSubdivision                        1.6.2.2    Autotask
Function        Get-AtwsClientPortalUser                           1.6.2.2    Autotask
Function        Get-AtwsContact                                    1.6.2.2    Autotask
Function        Get-AtwsContract                                   1.6.2.2    Autotask
Function        Get-AtwsContractBlock                              1.6.2.2    Autotask
Function        Get-AtwsContractCost                               1.6.2.2    Autotask
Function        Get-AtwsContractMilestone                          1.6.2.2    Autotask
Function        Get-AtwsContractRetainer                           1.6.2.2    Autotask
Function        Get-AtwsContractTicketPurchase                     1.6.2.2    Autotask
Function        Get-AtwsCountry                                    1.6.2.2    Autotask
Function        Get-AtwsCurrency                                   1.6.2.2    Autotask
Function        Get-AtwsExpenseItem                                1.6.2.2    Autotask
Function        Get-AtwsExpenseReport                              1.6.2.2    Autotask
Function        Get-AtwsInstalledProduct                           1.6.2.2    Autotask
Function        Get-AtwsInternalLocation                           1.6.2.2    Autotask
Function        Get-AtwsInvoiceTemplate                            1.6.2.2    Autotask
Function        Get-AtwsNotificationHistory                        1.6.2.2    Autotask
Function        Get-AtwsOpportunity                                1.6.2.2    Autotask
Function        Get-AtwsPriceListWorkTypeModifier                  1.6.2.2    Autotask
Function        Get-AtwsProduct                                    1.6.2.2    Autotask
Function        Get-AtwsProject                                    1.6.2.2    Autotask
Function        Get-AtwsProjectCost                                1.6.2.2    Autotask
Function        Get-AtwsProjectNote                                1.6.2.2    Autotask
Function        Get-AtwsPurchaseApproval                           1.6.2.2    Autotask
Function        Get-AtwsPurchaseOrder                              1.6.2.2    Autotask
Function        Get-AtwsQuote                                      1.6.2.2    Autotask
Function        Get-AtwsQuoteItem                                  1.6.2.2    Autotask
Function        Get-AtwsQuoteTemplate                              1.6.2.2    Autotask
Function        Get-AtwsResource                                   1.6.2.2    Autotask
Function        Get-AtwsResourceRole                               1.6.2.2    Autotask
Function        Get-AtwsResourceRoleQueue                          1.6.2.2    Autotask
Function        Get-AtwsResourceSkill                              1.6.2.2    Autotask
Function        Get-AtwsSalesOrder                                 1.6.2.2    Autotask
Function        Get-AtwsService                                    1.6.2.2    Autotask
Function        Get-AtwsServiceBundle                              1.6.2.2    Autotask
Function        Get-AtwsServiceCall                                1.6.2.2    Autotask
Function        Get-AtwsSkill                                      1.6.2.2    Autotask
Function        Get-AtwsSubscription                               1.6.2.2    Autotask
Function        Get-AtwsTask                                       1.6.2.2    Autotask
Function        Get-AtwsTaskNote                                   1.6.2.2    Autotask
Function        Get-AtwsTicket                                     1.6.2.2    Autotask
Function        Get-AtwsTicketCategory                             1.6.2.2    Autotask
Function        Get-AtwsTicketCategoryFieldDefaults                1.6.2.2    Autotask
Function        Get-AtwsTicketChecklistItem                        1.6.2.2    Autotask
Function        Get-AtwsTicketCost                                 1.6.2.2    Autotask
Function        Get-AtwsTicketNote                                 1.6.2.2    Autotask
Function        Get-AtwsTimeEntry                                  1.6.2.2    Autotask
Function        Get-AtwsUserDefinedFieldDefinition                 1.6.2.2    Autotask
Function        Get-AtwsWorkTypeModifier                           1.6.2.2    Autotask
Function        New-AtwsAccount                                    1.6.2.2    Autotask
Function        New-AtwsAccountAlert                               1.6.2.2    Autotask
Function        New-AtwsAccountNote                                1.6.2.2    Autotask
Function        New-AtwsAccountToDo                                1.6.2.2    Autotask
Function        New-AtwsActionType                                 1.6.2.2    Autotask
Function        New-AtwsBusinessLocation                           1.6.2.2    Autotask
Function        New-AtwsClientPortalUser                           1.6.2.2    Autotask
Function        New-AtwsContact                                    1.6.2.2    Autotask
Function        New-AtwsContract                                   1.6.2.2    Autotask
Function        New-AtwsContractBlock                              1.6.2.2    Autotask
Function        New-AtwsContractCost                               1.6.2.2    Autotask
Function        New-AtwsContractMilestone                          1.6.2.2    Autotask
Function        New-AtwsContractRetainer                           1.6.2.2    Autotask
Function        New-AtwsContractTicketPurchase                     1.6.2.2    Autotask
Function        New-AtwsExpenseItem                                1.6.2.2    Autotask
Function        New-AtwsExpenseReport                              1.6.2.2    Autotask
Function        New-AtwsInstalledProduct                           1.6.2.2    Autotask
Function        New-AtwsOpportunity                                1.6.2.2    Autotask
Function        New-AtwsProduct                                    1.6.2.2    Autotask
Function        New-AtwsProject                                    1.6.2.2    Autotask
Function        New-AtwsProjectCost                                1.6.2.2    Autotask
Function        New-AtwsProjectNote                                1.6.2.2    Autotask
Function        New-AtwsPurchaseOrder                              1.6.2.2    Autotask
Function        New-AtwsQuote                                      1.6.2.2    Autotask
Function        New-AtwsQuoteItem                                  1.6.2.2    Autotask
Function        New-AtwsResourceRoleQueue                          1.6.2.2    Autotask
Function        New-AtwsService                                    1.6.2.2    Autotask
Function        New-AtwsServiceBundle                              1.6.2.2    Autotask
Function        New-AtwsServiceCall                                1.6.2.2    Autotask
Function        New-AtwsServiceCallTicket                          1.6.2.2    Autotask
Function        New-AtwsServiceCallTicketResource                  1.6.2.2    Autotask
Function        New-AtwsSubscription                               1.6.2.2    Autotask
Function        New-AtwsTask                                       1.6.2.2    Autotask
Function        New-AtwsTaskNote                                   1.6.2.2    Autotask
Function        New-AtwsTicket                                     1.6.2.2    Autotask
Function        New-AtwsTicketChecklistItem                        1.6.2.2    Autotask
Function        New-AtwsTicketCost                                 1.6.2.2    Autotask
Function        New-AtwsTicketNote                                 1.6.2.2    Autotask
Function        New-AtwsTimeEntry                                  1.6.2.2    Autotask
Function        New-AtwsUserDefinedFieldDefinition                 1.6.2.2    Autotask
Function        Remove-AtwsAccountToDo                             1.6.2.2    Autotask
Function        Remove-AtwsActionType                              1.6.2.2    Autotask
Function        Remove-AtwsContractCost                            1.6.2.2    Autotask
Function        Remove-AtwsProjectCost                             1.6.2.2    Autotask
Function        Remove-AtwsQuoteItem                               1.6.2.2    Autotask
Function        Remove-AtwsServiceBundle                           1.6.2.2    Autotask
Function        Remove-AtwsServiceCall                             1.6.2.2    Autotask
Function        Remove-AtwsSubscription                            1.6.2.2    Autotask
Function        Remove-AtwsTicketChecklistItem                     1.6.2.2    Autotask
Function        Remove-AtwsTicketCost                              1.6.2.2    Autotask
Function        Set-AtwsAccount                                    1.6.2.2    Autotask
Function        Set-AtwsAccountAlert                               1.6.2.2    Autotask
Function        Set-AtwsAccountNote                                1.6.2.2    Autotask
Function        Set-AtwsAccountToDo                                1.6.2.2    Autotask
Function        Set-AtwsActionType                                 1.6.2.2    Autotask
Function        Set-AtwsBillingItem                                1.6.2.2    Autotask
Function        Set-AtwsBusinessLocation                           1.6.2.2    Autotask
Function        Set-AtwsClientPortalUser                           1.6.2.2    Autotask
Function        Set-AtwsContact                                    1.6.2.2    Autotask
Function        Set-AtwsContract                                   1.6.2.2    Autotask
Function        Set-AtwsContractBlock                              1.6.2.2    Autotask
Function        Set-AtwsContractCost                               1.6.2.2    Autotask
Function        Set-AtwsContractMilestone                          1.6.2.2    Autotask
Function        Set-AtwsContractRetainer                           1.6.2.2    Autotask
Function        Set-AtwsContractTicketPurchase                     1.6.2.2    Autotask
Function        Set-AtwsCountry                                    1.6.2.2    Autotask
Function        Set-AtwsCurrency                                   1.6.2.2    Autotask
Function        Set-AtwsExpenseItem                                1.6.2.2    Autotask
Function        Set-AtwsExpenseReport                              1.6.2.2    Autotask
Function        Set-AtwsInstalledProduct                           1.6.2.2    Autotask
Function        Set-AtwsOpportunity                                1.6.2.2    Autotask
Function        Set-AtwsPriceListWorkTypeModifier                  1.6.2.2    Autotask
Function        Set-AtwsProduct                                    1.6.2.2    Autotask
Function        Set-AtwsProject                                    1.6.2.2    Autotask
Function        Set-AtwsProjectCost                                1.6.2.2    Autotask
Function        Set-AtwsProjectNote                                1.6.2.2    Autotask
Function        Set-AtwsPurchaseApproval                           1.6.2.2    Autotask
Function        Set-AtwsPurchaseOrder                              1.6.2.2    Autotask
Function        Set-AtwsQuote                                      1.6.2.2    Autotask
Function        Set-AtwsQuoteItem                                  1.6.2.2    Autotask
Function        Set-AtwsResource                                   1.6.2.2    Autotask
Function        Set-AtwsResourceRoleQueue                          1.6.2.2    Autotask
Function        Set-AtwsResourceSkill                              1.6.2.2    Autotask
Function        Set-AtwsSalesOrder                                 1.6.2.2    Autotask
Function        Set-AtwsService                                    1.6.2.2    Autotask
Function        Set-AtwsServiceBundle                              1.6.2.2    Autotask
Function        Set-AtwsServiceCall                                1.6.2.2    Autotask
Function        Set-AtwsSubscription                               1.6.2.2    Autotask
Function        Set-AtwsTask                                       1.6.2.2    Autotask
Function        Set-AtwsTaskNote                                   1.6.2.2    Autotask
Function        Set-AtwsTicket                                     1.6.2.2    Autotask
Function        Set-AtwsTicketCategory                             1.6.2.2    Autotask
Function        Set-AtwsTicketChecklistItem                        1.6.2.2    Autotask
Function        Set-AtwsTicketCost                                 1.6.2.2    Autotask
Function        Set-AtwsTicketNote                                 1.6.2.2    Autotask
Function        Set-AtwsTimeEntry                                  1.6.2.2    Autotask
Function        Set-AtwsUserDefinedFieldDefinition                 1.6.2.2    Autotask
Function        Set-AtwsWorkTypeModifier                           1.6.2.2    Autotask

```

**Comment by klemmestad** [comment]
Hi, this will/should/must be fixed quickly. I expect to upload a new module version today.

**Comment by klemmestad** [comment]
Should be fixed in module version 1.6.2.8
